### PR TITLE
EUI-3777 create prepended tabs on case view v2

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3777

### Change description ###
Moved the caseviewer route from toolkit to MC
Added to extra container components which will house the tasks and roles and access tab.

**Does this PR introduce a breaking change?** (check one with "x")

```
[X] Yes
[ ] No
```

### CODE REVIEW

File: src/app/services/ccd-config/ccd-case.config.ts [hunk 1]
```diff
@@ -148,4 +158,38 @@ export class AppConfig extends AbstractAppConfig {
...
+        (isFeatureEnabled) =>
+          this.workallocationUrl = AppUtils.getFeatureToggledUrl(
+            isFeatureEnabled,
+            WorkAllocationTaskService.WorkAllocationUrl
+          )
+      );
+  }
+
+  private enablePrependedTabs(feature: string, userDetails: UserDetails): boolean {
+    return feature === 'WorkAllocationRelease2'
```
Comment (udaydenduluri33) [inline src/app/services/ccd-config/ccd-case.config.ts:192]: Please refer constants and string literal

File: src/cases/case-feature.routes.ts [hunk 2]
```diff
@@ -102,7 +105,26 @@ export const ROUTES: Routes = [
...
+                {
+                  path: '',
+                  pathMatch: 'full',
+                },
+                {
+                  path: 'tasks',
+                  component: TasksContainerComponent
+                },
+                {
+                  path: 'roles and access',
```
Comment (udaydenduluri33) [inline src/cases/case-feature.routes.ts:122]: remove space in the path

File: src/app/services/ccd-config/ccd-case.config.ts [hunk 3]
```diff
@@ -148,4 +158,38 @@ export class AppConfig extends AbstractAppConfig {
...
+    this.featureToggleService
+      .isEnabled(AppConstants.FEATURE_NAMES.workAllocation)
+      .subscribe(
+        (isFeatureEnabled) =>
+          this.workallocationUrl = AppUtils.getFeatureToggledUrl(
+            isFeatureEnabled,
+            WorkAllocationTaskService.WorkAllocationUrl
+          )
+      );
+  }
```
Comment (rayliangatos) [inline src/app/services/ccd-config/ccd-case.config.ts:189]: Could these move to the container(exui container) level? 
The problem to put here is ccd-case.config.ts is intantiated/implemented when the APP is started, we shouldn't put the container's dynamic logic here.
The ccd-case.config.ts is used for put all those configuration(constant immutable value) settings.

**Review by rayliangatos — CHANGES_REQUESTED** [review]
leave request change comment

File: src/app/services/ccd-config/ccd-case.config.ts (hunk 3 above)
Comment (kuda1992) [reply src/app/services/ccd-config/ccd-case.config.ts]: @rayliangatos I cannot move this to container level. As this config needs to be passed to the `toolkit`. As well as its according to the design. 
https://tools.hmcts.net/confluence/display/EUI/CCD+Toolkit+-+Custom+added+tabs

**Review by rayliangatos — APPROVED** [review]
As discussed, we will probably refactor the current design in next IP week, for timebeing approve the PR.

**Review by connorpgpmcelroy — APPROVED** [review]
Can see comments have been addresses and no issues with code seen from me

**Pull request merged**